src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$hover-color: #00acdc;
+
+$transition: 0.3s;
+
+$white-color: #fff;
+
+$column-count: 1023.98px 3, 767.98px 2, 487.98px 1;
```

Comment:
If the styles have fractional values of sizes, then something is wrong. All sizes must be whole values.